src/components/Footer/Footer.js

```diff
@@ -1,7 +1,155 @@
 import React from 'react';
 
+import styled from 'styled-components';
+
 function Footer() {
-  return <div>Footer</div>;
+  return (
+    <CommonFooter>
+      <InnerBox>
+        <InfoContainer>
+          <ul className="enterInfo">
+            <Divider>
+              <li>이용안내</li>
+              <li>인재채용</li>
+              <li>기술블로그</li>
+            </Divider>
+            <Divider>
+              <li>윤리강령</li>
+              <li>이용약관</li>
+              <li>개인정보 처리 방침</li>
+            </Divider>
+            <Divider>
+              <li>전자금융거래 약관</li>
+            </Divider>
+          </ul>
+          <Service>
+            <ServiceItem>
+              <span>
+                <b>고객센터</b>평일10시~18시 (점심12시~13시)
+              </span>
+            </ServiceItem>
+            <ServiceItem>
+              <span>
+                전화<b>02-0000-0000</b>
+              </span>
+              <span>
+                팩스<b>02-0000-0000</b>
+              </span>
+            </ServiceItem>
+            <ServiceItem>
+              <span>
+                대표메일<b>ask@22percent.kr</b>
+              </span>
+              <span>
+                사업제휴<b>partner@22percent.kr</b>
+              </span>
+            </ServiceItem>
+          </Service>
+          <SnsBox>
+            <i className="fab fa-facebook"></i>
+            <i class="fab fa-google-plus"></i>
+            <i className="fab fa-snapchat"></i>
+          </SnsBox>
+        </InfoContainer>
+        <DescContainer>
+          <div className="companyInfo">
+            <span>(주)에잇퍼센트 | 대표 </span>
+            <span>
+              법인등록번호 | 온라인투자연계금융업 | 서울특별시 강남구 삼성동
+              테헤란로 427
+            </span>
+          </div>
+          <div className="serviceAlert">
```

Comment:
```suggestion
          <ServiceAlert>
```
클래스 네임 붙일 이름으로 스타일드 컴포넌트를 만들어주면 어떨까요?